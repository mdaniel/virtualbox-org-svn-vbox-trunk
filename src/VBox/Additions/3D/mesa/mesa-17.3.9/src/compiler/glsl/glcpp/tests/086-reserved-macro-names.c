#define __BAD reserved
#define GL_ALSO_BAD() also reserved
#define THIS__TOO__IS__BAD reserved
