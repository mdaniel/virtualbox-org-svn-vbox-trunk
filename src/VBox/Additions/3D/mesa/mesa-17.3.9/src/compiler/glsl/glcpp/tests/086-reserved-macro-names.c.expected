0:1(9): preprocessor warning: Macro names containing "__" are reserved for use by the implementation.

0:2(9): preprocessor error: Macro names starting with "GL_" are reserved.

0:3(9): preprocessor warning: Macro names containing "__" are reserved for use by the implementation.




